web/i18n/language.ts

```diff
@@ -25,93 +25,115 @@ export const languages = [
   {
     value: 'en-US',
     name: 'English (United States)',
+    prompt_name: 'English',
     example: 'Hello, Dify!',
     supported: true,
   },
   {
     value: 'zh-Hans',
     name: '简体中文',
+    prompt_name: 'Chinese',
```

Comment:
Not sure if both should be Chinese. I think this is the Simplified one and the other is Traditional?